credentials.example.yml

```diff
@@ -10,3 +10,8 @@ rds-staging-username:
 rds-staging-password:
 rds-development-username:
 rds-development-password:
+
+slack-webhook-url:
+slack-channel:
+slack-username:
+slack-icon-url:
```

Comment:
We need to share this information with folks like @erik-burgess just like the RDS credentials, right?